build/deb_rpm/v2-jf/build-scripts/pack.sh

```diff
@@ -159,10 +158,10 @@ rpmSign()(
  	if [[ -f "${filePath}" && -f "${gpgFileInHost}" ]]; then
  		log ""; log "";
  		log "Initiating rpm sign on ${filePath}..."
- 		docker run --rm --name cli-rpm-sign -v "${filePath}":${filePathInImage} \
+ 		docker run --rm --name cli-rpm-sign -v "${filePath}"":${filePathInImage}" \
```

Comment:
Same for here:
```suggestion
 		docker run --rm --name cli-rpm-sign -v "${filePath}:${filePathInImage}" \
```

* I saw other places whereby there are redundant quotes